config/config.go

```diff
@@ -0,0 +1,43 @@
+package config
```

Comment:
Correct me if I am wrong but assumption is that setting this up should let me use `config.Config.Environment` to get the environment the code is running in.